src/main/java/muramasa/antimatter/capability/machine/MachineRecipeHandler.java

```diff
@@ -193,16 +191,12 @@ public void onMachineEvent(IMachineEvent event, Object... data) {
             tile.markDirty(); //TODO determine if needed
         }
         if (event instanceof MachineEvent) {
-            switch ((MachineEvent)event) {
-                case ENERGY_INPUTTED:
-                    if (this.tile.getMachineState() == IDLE)
-                        //NO_POWER is bad name i guess, by this i mean try to do a recipe check next tick.
-                        this.tile.setMachineState(NO_POWER);
-                    if (this.tile.getMachineState() == POWER_LOSS)
-                        this.tile.setMachineState(ACTIVE);
-                    break;
-                default:
-                    break;
+            if (event == MachineEvent.ENERGY_INPUTTED) {
+                if (this.tile.getMachineState() == IDLE)
+                    //NO_POWER is bad name i guess, by this i mean try to do a recipe check next tick.
+                    this.tile.setMachineState(NO_POWER);
+                if (this.tile.getMachineState() == POWER_LOSS)
+                    this.tile.setMachineState(ACTIVE);
```

Comment:
It's better to break here, as all the conditions would follow through + checked.

Reply:
Done